stackanetes-services/mariadb/mariadb-galera.yml

```diff
@@ -0,0 +1,68 @@
+type: custom/mariadb-galera
```

Comment:
This file should replace existing mariadb.yml without this manage_all will deploy both of them.